src/containers/Header/style.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const HeaderWrapper = styled.header`
+  display: flex;
+  justify-content: space-between;
+  background-color: #110F10;
+  align-items: center;
+`;
+
+export const SearchWrapper = styled.div`
```

Comment:
If you do not apply any styles, you can use regular `div` inside of your component